modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/UpNextQueueImpl.kt

```diff
@@ -205,8 +205,9 @@ class UpNextQueueImpl @Inject constructor(
 
         val mutableList = episodes.toMutableList()
         if (isEmpty) {
-            playNextNow(mutableList.first(), downloadManager, null)
-            mutableList.removeFirst()
+            val firstEpisode = mutableList.first()
+            playNextNow(firstEpisode, downloadManager, null)
+            mutableList.remove(firstEpisode)
```

Comment:
This fixes the following test issue:
> Call requires API level 35 (current min is 24): java. util. List#removeFirst (Prior to API level 35, this call would resolve to a Kotlin stdlib extension function.

Reply:
Alternatively, we can set `"-Xjdk-release=$javaVersion"` flag so we can use Kotlin APIs.

https://jakewharton.com/kotlins-jdk-release-compatibility-flag